Automattic/jetpack
# Add step by step instructions to troubleshoot critical CSS issues

Fixes #20682 

#### Changes proposed in this Pull Request:
Broken down critical CSS suggestions into smaller steps to make it easier for users to follow.

#### Jetpack product discussion
pbIAo1-kG-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
Generate appropriate errors through MU plugins to simulate each HTTP error. For example:
```
if ( ! empty( $_GET['jb-generate-critical-css' ] ) ) {
	if ( strpos( $_SERVER['REQUEST_URI'], 'hello-world' ) !== false ) {
		header("HTTP/1.0 404 Not Found");
		die();
	}
}
```
To generate a 404 error.

Each other type of error requires specific elaborate steps to generate so I am not including them here. But you can check the screenshots at: pbIAo1-kG-p2#comment-755.
